sdk/index.ts

```diff
@@ -169,47 +192,55 @@ export class MidcontractProtocol {
     });
   }
 
-  private get escrowPayeeFee(): Promise<number> {
-    return this.public
-      .readContract({
-        functionName: "payeeFee",
-        address: this.escrow,
-        abi: escrow,
-      })
-      .then(pf => this.escrowDenominatorFee.then(df => Number(pf) / Number(df)));
-  }
+  // private get escrowClientFee(): Promise<number> {
+  //   return this.public
+  //     .readContract({
+  //       functionName: "feeClient",
+  //       address: this.escrow,
+  //       abi: escrow,
+  //     })
+  //     .then(pf => this.escrowDenominatorFee.then(df => Number(pf) / Number(df)));
+  // }
+  //
+  // private get escrowContractorFee(): Promise<number> {
+  //   return this.public
+  //     .readContract({
+  //       address: this.escrow,
+  //       abi: escrow,
+  //       functionName: "feeContractor",
+  //     })
+  //     .then(rf => this.escrowDenominatorFee.then(df => Number(rf) / Number(df)));
+  // }
+
+  // private get escrowDenominatorFee(): Promise<bigint> {
+  //   return this.public.readContract({
+  //     address: this.escrow,
+  //     abi: escrow,
+  //     functionName: "FEE_DENOMINATOR",
+  //   });
+  // }
+
+  async escrowDepositAmount(amount: bigint, clientAddress: Address, feeConfig: FeeConfig): Promise<DepositAmount> {
```

Comment:
Input parameters and return values of numbers should be of numeric type, we use bigint type only in the implementation, the essence of the SDK is to simplify such things and provide a simple way to work, and not make us think about data conversion, this is something we can avoid at the client level